src/metrics.jl

```diff
@@ -340,21 +340,21 @@ end
 """
     evaluation_metrics(args...; optimal_threshold_class=nothing, kwargs...)
 
-Return [`evaluation_metrics_row`](@ref) after converting output `EvaluationRow`
-into a `Dict`. For argument details, see [`evaluation_metrics_row`](@ref).
+Return [`evaluation_metrics_record`](@ref) after converting output `EvaluationV1`
+into a `Dict`. For argument details, see [`evaluation_metrics_record`](@ref).
 """
 function evaluation_metrics(args...; optimal_threshold_class=nothing, kwargs...)
-    row = evaluation_metrics_row(args...;
-                                 optimal_threshold_class=something(optimal_threshold_class,
-                                                                   missing), kwargs...)
-    return _evaluation_row_dict(row)
+    row = evaluation_metrics_record(args...;
+                                    optimal_threshold_class=something(optimal_threshold_class,
+                                                                      missing), kwargs...)
+    return _evaluation_dict(row)
 end
 
 """
-    evaluation_metrics_row(observation_table, classes, thresholds=0.0:0.01:1.0;
+    evaluation_metrics_record(observation_table, classes, thresholds=0.0:0.01:1.0;
                            strata::Union{Nothing,AbstractVector{Set{T}} where T}=nothing,
                            optimal_threshold_class::Union{Missing,Nothing,Integer}=missing)
-    evaluation_metrics_row(predicted_hard_labels::AbstractVector,
+    evaluation_metrics_record(predicted_hard_labels::AbstractVector,
```

Comment:
even though this isn't exported, it is documented, so it seems like a good idea to provide a deprecation path (at least throw an error that points to the new method).